src/libraries/Common/src/System/Net/Security/SslClientAuthenticationOptionsExtensions.cs

```diff
@@ -19,6 +19,7 @@ public static SslClientAuthenticationOptions ShallowClone(this SslClientAuthenti
                 AllowRenegotiation = options.AllowRenegotiation,
                 ApplicationProtocols = options.ApplicationProtocols != null ? new List<SslApplicationProtocol>(options.ApplicationProtocols) : null,
                 CertificateRevocationCheckMode = options.CertificateRevocationCheckMode,
+                CertificateChainPolicy = options.CertificateChainPolicy,
```

Comment:
Are we missing a test that should have failed without this?

Reply:
yes, we should add test to HTTP once QUIC supports CertificateChainPolicy.